sensorlib/config/noise_model_config.yaml

```diff
@@ -1,4 +1,9 @@
 noise_model_name: GaussianNoiseModel
+stages:
+    position_noise: True
+    orientation_noise: True
+    type_noise: True
```

Comment:
We talked about defaulting to turn of type noise since its implementation is incomplete currently right. Last we talked I remember this noise essentially assigning a random type classification for each detection which we concluded is not really noise and more of just invalid data. Noise would be introducing some error to an otherwise valid signal. Making the entire signal random error is not really noise. Let me know if anything has changed.

Reply:
False by default now. This is a hook for future robust implementation.